mix.exs

```diff
@@ -70,6 +70,7 @@ defmodule Tesla.Mixfile do
       # other
       {:fuse, "~> 2.4", optional: true},
       {:telemetry, "~> 0.4 or ~> 1.0", optional: true},
+      {:mox, "~> 1.0.0", optional: true},
```

Comment:
Any reason you're not pointing to latest `mox` (1.2.0)? The previous `mix.lock` was resolving to this, too